pkg/restapi/issuer/operation/operations.go

```diff
@@ -46,6 +46,9 @@ const (
 	vcsProfileCookie = "vcsProfile"
 
 	issueCredentialURLFormat = "%s/%s" + "/credentials/issueCredential"
+
+	// contexts
+	trustBlocExampleContext = "https://trustbloc.github.io/context/vc/examples-ext-v1.jsonld"
```

Comment:
Ideally we should have a mapping between scope and the context. Currently, one context supports all the scopes.